python/neuroglancer/pipeline/storage.py

```diff
@@ -107,6 +117,50 @@ def get_file(self, file_path):
             content = self._maybe_uncompress(content)
         return content
 
+    def get_files(self, file_paths):
+        """
+        returns a list of files faster by using threads
+        """
+
+        file_paths = deque(file_paths)
+
+        results = []
+
+        def store_result(path, result, error):
```

Comment:
I disagree on this one. This method is only useful in this context. Maybe it's less common in python, but in Javascript, closures, anonymous functions and others are used all the time. Python is a little overly restrictive in how it allows you to define functions. Placing a one time use function in the class scope makes it harder to understand because now there's a possibility it might be used elsewhere.